webapp/views.py

```diff
@@ -398,7 +398,7 @@ def advantage_view():
     new_subscription_id = None
     open_subscription = flask.request.args.get("subscription", None)
     stripe_publishable_key = os.getenv(
-        "STRIPE_PUBLISHABLE_KEY", "pk_live_68aXqowUeX574aGsVck8eiIE"
+        "STRIPE_PUBLISHABLE_KEY", "pk_test_yndN9H0GcJffPe0W58Nm64cM00riYG4N46"
```

Comment:
It seems weird to be hardcoding the default from `os.getenv` here?

Reply:
I'm not too fussed. Yeah it could be in `.env` and then also written into Kubernetes config, but that way it is a tiny bit of a pain 'cos it's kept in at least 2 places. I think whether we hard-code it depends on whether it needs to change between environments, and how we're gonna want to update it. If it needs to be different between dev / staging / prod then it can't be hard coded. Or if we might want to change the publishable key without doing a release, then k8s configs would allow us to do that.

If neither of those are the case - it's easiest for it to be the same between all environments, and we are perfectly happy to update it through PR + release, then I'm actually wondering if there's any point using `os.getenv` at all. Maybe we should just do `stripe_publishable_key = "pk_test_yndN9H0GcJffPe0W58Nm64cM00riYG4N46"` for now, as the most straightforward, declarative solution, and only add in the ability to read it from an environment variable as and when we actually need it.